### PR TITLE
Modify README of bert example

### PR DESCRIPTION
**Pull request by chenmoneygithub**
1) Fix the curl command, and add curl instruction for vocab downloading.
2) Add output flag for glue_fine_tuning script.

### CODE REVIEW

File: examples/bert/README.md [hunk 1]
```diff
@@ -79,7 +79,8 @@ The latest wikipedia dump can be downloaded [at this link](https://dumps.wikimed
 or via command line:
 
 ```shell
-curl https://dumps.wikimedia.org/enwiki/latest/enwiki-latest-pages-articles.xml.bz2
+curl https://dumps.wikimedia.org/enwiki/latest/enwiki-latest-pages-articles.xml.bz2 \
```
Comment (mattdangerw) [inline examples/bert/README.md:82]: I think you can do `curl -O url` without needing to duplicate the filename.

File: examples/bert/README.md [hunk 2]
```diff
@@ -126,7 +127,12 @@ The `create_vocabulary.py` script allows you to compute your own WordPiece
 vocabulary for use with BERT. In most cases however, it is desirable to use the
 standard BERT vocabularies from the original models. You can download the
 English uncased vocabulary
-[here](https://storage.googleapis.com/tensorflow/keras-nlp/examples/bert/bert_vocab_uncased.txt).
+[here](https://storage.googleapis.com/tensorflow/keras-nlp/examples/bert/bert_vocab_uncased.txt),
+or in your terminal run:
+```shell
```
Comment (mattdangerw) [inline examples/bert/README.md:132]: newline before codeblock.

File: examples/bert/README.md [hunk 3]
```diff
@@ -126,7 +127,12 @@ The `create_vocabulary.py` script allows you to compute your own WordPiece
...
+curl https://storage.googleapis.com/tensorflow/keras-nlp/examples/bert/bert_vocab_uncased.txt \
```
Comment (mattdangerw) [inline examples/bert/README.md:133]: same here `curl -O url` to keep this a one liner.

File: examples/bert/README.md [hunk 4]
```diff
@@ -220,7 +226,8 @@ training for a few epochs to finetune the model.
 
 ```shell
 python3 examples/bert/run_glue_finetuning.py \
-    --saved_model_input path/to/model/ \
```
Comment (mattdangerw) [inline examples/bert/README.md:223]: Can probably just leave this as is?

We don't really need to the output model from fine-tuning to be saved anywhere usually. Really what we would want is to save some predictions in the GLUE submission format (tracked here https://github.com/keras-team/keras-nlp/issues/115).

**Review by mattdangerw — APPROVED** [review]
Looks good! Few nits.

File: examples/bert/README.md (hunk 3 above)
Comment (chenmoneygithub) [reply examples/bert/README.md:132]: I am doing a renaming here - in the README we are mixing vocab.txt with bert_vocab_uncased.txt, so to unify them I am only keeping vocab.txt at every reference.

File: examples/bert/README.md (hunk 4 above)
Comment (chenmoneygithub) [reply examples/bert/README.md:223]: Yea, I feel that users may find the need to reevaluate the model? If we do not save the model then people won't have access to the finetuned model except finetuning again.

File: examples/bert/README.md [hunk 5]
```diff
@@ -79,7 +79,7 @@ The latest wikipedia dump can be downloaded [at this link](https://dumps.wikimed
 or via command line:
 
 ```shell
-curl https://dumps.wikimedia.org/enwiki/latest/enwiki-latest-pages-articles.xml.bz2
+curl -O https://dumps.wikimedia.org/enwiki/latest/enwiki-latest-pages-articles.xml.bz2
```
Comment (mattdangerw) [inline examples/bert/README.md:82]: looks like trailing space here.

**Review by mattdangerw — APPROVED** [review]

File: examples/bert/README.md (hunk 5 above)
Comment (chenmoneygithub) [reply examples/bert/README.md]: good catch!

**Pull request merged**